Shared/AppInfo.swift

```diff
@@ -54,6 +54,10 @@ class AppInfo {
     open class func isSimulator() -> Bool {
         return ProcessInfo.processInfo.environment["SIMULATOR_ROOT"] != nil
     }
+    
+    open class func isTesting() -> Bool {
+        return ProcessInfo.processInfo.arguments.contains("testMode")
```

Comment:
Nice! This will change slightly when I land this: https://github.com/mozilla-mobile/focus-ios/pull/923/commits/c6c6bb7f50fc5a18b5080546d32c0b470715f82a#diff-e350d026dbb7dba45281e61e0f81a18fR62
However I'll make sure not to conflict with this test.